### PR TITLE
Add new docs for "Billing & Invoice FAQ"

### PR DESCRIPTION
**Pull request by AhyoungRyu**
I added new docs for "Billing & Invoice FAQ". Here is the screenshot images :)

screen shot 2016-07-21 at 3 53 12 pm
screen shot 2016-07-21 at 3 53 35 pm
unsubscribe
screen shot 2016-07-21 at 3 53 48 pm
get_subscription
screen shot 2016-07-21 at 3 53 56 pm

Please point me if there are sth wrong or need more explanation.

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
Also added new section for this to [http://help.zeppelinhub.com/support/](http://help.zeppelinhub.com/support/) 
screen shot 2016-07-21 at 3 55 39 pm

**Comment by AhyoungRyu** [comment]
\cc @anthonycorbacho @minahlee Could you review this one?

File: docs/payment_issue.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<span class="header-font">Billing & Invoicing FAQ</span>
+
+This page will help you better understand and manage your service plan & billing statements.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: This page will help you to understand how to manage your ZeppelinHub Business subscription.

File: docs/payment_issue.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+If you have any questions which is not in the below common questions, please do not hesitate to contact us [info@nflabs.com](mailto:info@nflabs.com).
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: IIf you have any questions which are not in this FAQ, please do not hesitate to conta.....

File: docs/payment_issue.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ZeppelinHub support team will be happy to help you.
+
+<br/>
+## <span class="support-font">1. Can I use ZeppelinHub for FREE?</span>
+
+Yes, you can use ZeppelinHub for free.
+We basically aim to provide ZeppelinHub service for free to use, share and collaborate with as many people you want for as long as you want.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: ZeppelinHub Freemium plan provides unlimited share, notebook backups and collaboration with anyone publicly and privately.

File: docs/payment_issue.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+For the further details about available features for **FREE** tier users, see **FREE Card** in [ZeppelinHub Pricing Guide](https://www.zeppelinhub.com/pricing).
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: For further details about available features in the **Business** plan, please check the [ZeppelinHub Pricing Page](https://www.zeppelinhub.com/pricing).

File: docs/payment_issue.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+<img src="../img/unsubscribe.gif" class="image-box big-img"/>
+
+## <span class="support-font">7. What's happend if I add/cancel the license?</span>
+
+You can subscribe regularly not only for you but for your team members. 
+When you get a subscription for the first time or after then, you can buy more licenses for your team members in the billing page.
+
+<img src="../img/get_subscription.gif" class="image-box big-img"/>
+
+After the payment, you can select users for each licenses. Of course it's possible to cancel the licenses as well if you want.
```
Comment (khalidhuseynov) [inline docs/payment_issue.md]: 1. `for each of the licences`
2. not sure whether need `if you want` at the end :)

File: docs/payment_issue.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+<br/>
+## <span class="support-font">1. Can I use ZeppelinHub for FREE?</span>
+
+Yes, you can use ZeppelinHub for free.
+We basically aim to provide ZeppelinHub service for free to use, share and collaborate with as many people you want for as long as you want.
+For the further details about available features for **FREE** tier users, see **FREE Card** in [ZeppelinHub Pricing Guide](https://www.zeppelinhub.com/pricing).
+
+## <span class="support-font">2. How can I check or upgrade my service plan?</span>
+
+You can manage all of your payment and subscription information in the **Settings -> Billing** page.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: You can manage your subscription in the **Settings -> Billing** page.

File: docs/payment_issue.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="../img/dropdown_setting.png" class="image-box small-img"/>
+
+You can check your current service plan below the profile picture and directly go to service plan upgrading page.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: If you have a **Freemium Plan**, you will have a link to upgrade to a **Business Plan** in the profile page.

File: docs/payment_issue.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="../img/dropdown_setting.png" class="image-box small-img"/>
+
+You can check your current service plan below the profile picture and directly go to service plan upgrading page. 
+
+<img src="../img/profile.png" class="image-box big-img"/>
+
+## <span class="support-font">3. What are my payment options?</span>
+
+You can use your credit card to pay for any service plan.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: Right now, we only support payment with Visa Card, MasterCard and American Express Card.

File: docs/payment_issue.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+Once you get your subscription, automatically charged for every months until you cancel the payment.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: Once you get your subscription, you will be automatically charged every month.

File: docs/payment_issue.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+## <span class="support-font">4. Can I get an invoice for my payment?</span>
+
+We are using [Stripe](https://stripe.com/) as our backend payment infrastructure. Stripe will send an invoice to your email address after you paid.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: I think that we should not talk about stripe here, thats something the user doesn't care.

File: docs/payment_issue.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+<img src="../img/get_subscription.gif" class="image-box big-img"/>
+
+After the payment, you can select users for each licenses. Of course it's possible to cancel the licenses as well if you want. 
+The changed cost will be charged in the next month.
+
+<img src="../img/manage_payment.png" class="image-box big-img"/>
+
+## <span class="support-font">8. Payment ammount is not reflected right after I paid. What's going on?</span>
+
+We are using [Stripe](https://stripe.com/) as our payments infrastructure. Although your payment is successfully done, Stripe needs some time to process your payment.
```
Comment (anthonycorbacho) [inline docs/payment_issue.md]: Although your payment is successfully done, we will need couple of minute to validate the transaction.

File: docs/payment_issue.md (hunk 4 above)
Comment (AhyoungRyu) [inline docs/payment_issue.md]: It's about free users. You mean "free" plan?

Comment (anthonycorbacho) [inline docs/payment_issue.md]: No business, we already talk about free plan, we need to attract people to pay

**Comment by AhyoungRyu** [comment]
@anthonycorbacho @khalidhuseynov All the feedbacks are addressed :)

**Comment by AhyoungRyu** [comment]
@anthonycorbacho Is it okay to merge & deploy?

**Comment by anthonycorbacho** [comment]
@AhyoungRyu lets ask @corneadoug to read proof one time and then lets merge it with ❤️

File: docs/payment_issue.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
+<span class="header-font">Billing & Invoicing FAQ</span>
+
+This page will help you to understand how to manage your ZeppelinHub Business subscription.
+If you have any questions which is not in this FAQ, please do not hesitate to contact us [info@nflabs.com](mailto:info@nflabs.com).
```
Comment (corneadoug) [inline docs/payment_issue.md]: Contact us at info@nflabs.com

File: docs/payment_issue.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+ZeppelinHub support team will be happy to help you.
+
+<br/>
+## <span class="support-font">1. Can I use ZeppelinHub for FREE?</span>
+
+Yes, you can use ZeppelinHub for free.
+ZeppelinHub Freemium plan provides unlimited share, notebook backups and collaboration with anyone publicly and privately.
```
Comment (corneadoug) [inline docs/payment_issue.md]: unlimited notebook sharing, backup and collaboration

File: docs/payment_issue.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+For further details about available features in the **BUSINESS** plan, please check [ZeppelinHub Pricing Guide](https://www.zeppelinhub.com/pricing).
```
Comment (corneadoug) [inline docs/payment_issue.md]: please visit the ZeppelinHub Pricing Guide

File: docs/payment_issue.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+## <span class="support-font">4. Can I get an invoice for my payment?</span>
+
+We will send an invoice to your email address after you paid.
+You can check all of your invoices below the **Payment History** section as well.
+The email address can also be updated in the billing page.
+
+## <span class="support-font">5. I have a coupon. How can I use this?</span>
+
+We issued **FREE UPGRADING COUPONS** so that ZeppelinHub users can upgrade and explore **BUSINESS** tier features for one month without any payment.
+Just type the code in the coupon field when you get a subscription. But please note that you still need to enter your vaild credit card number.
```
Comment (corneadoug) [inline docs/payment_issue.md]: your vaild -> a valid

File: docs/payment_issue.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+Don't worry. We won't charge anything. 
+
+<span class="note-font">Since the free coupon is vaild only for one person, you might need to pay for other people if you want to pay for several team members.
```
Comment (corneadoug) [inline docs/payment_issue.md]: vaild only for one person -> valid for only one person

File: docs/payment_issue.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+Whenever you want to stop the payment, you can cancel the subscription in the billing page. It will be reflected right after the cancelation.
+Then you won't be charged from the next month.
+
+<img src="../img/unsubscribe.gif" class="image-box big-img"/>
+
+## <span class="support-font">7. What's happend if I add/cancel the license?</span>
+
+You can subscribe regularly not only for you but for your team members. 
+When you get a subscription for the first time or after then, you can buy more licenses for your team members in the billing page.
```
Comment (corneadoug) [inline docs/payment_issue.md]: You can add more licenses when you first get a subscription, or later in your Billing page.

File: docs/payment_issue.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+
+<img src="../img/get_subscription.gif" class="image-box big-img"/>
+
+After the payment, you can select users for each of the licenses. Of course it's possible to cancel the licenses as well.
```
Comment (corneadoug) [inline docs/payment_issue.md]: for each license.

File: docs/payment_issue.md [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+
+## <span class="support-font">7. What's happend if I add/cancel the license?</span>
+
+You can subscribe regularly not only for you but for your team members. 
+When you get a subscription for the first time or after then, you can buy more licenses for your team members in the billing page.
+
+<img src="../img/get_subscription.gif" class="image-box big-img"/>
+
+After the payment, you can select users for each of the licenses. Of course it's possible to cancel the licenses as well. 
+The changed cost will be charged in the next month.
```
Comment (corneadoug) [inline docs/payment_issue.md]: Changes of your billing will be charged the next month.

File: docs/payment_issue.md [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+
+<img src="../img/manage_payment.png" class="image-box big-img"/>
+
+## <span class="support-font">8. Payment ammount is not reflected right after I paid. What's going on?</span>
+
+We are using [Stripe](https://stripe.com/) as our payments infrastructure.
+Although your payment is successfully done, we will need couple of minutes to validate the transaction.
+Don't worry. It won't take too much time. Please refresh ZeppelinHub billing page after few minutes. Then it will show the updated **Total Payment Amount**.
```
Comment (corneadoug) [inline docs/payment_issue.md]: a few minutes

File: docs/payment_issue.md [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
+
+After the payment, you can select users for each of the licenses. Of course it's possible to cancel the licenses as well. 
+The changed cost will be charged in the next month.
+
+<img src="../img/manage_payment.png" class="image-box big-img"/>
+
+## <span class="support-font">8. Payment ammount is not reflected right after I paid. What's going on?</span>
+
+We are using [Stripe](https://stripe.com/) as our payments infrastructure.
+Although your payment is successfully done, we will need couple of minutes to validate the transaction.
```
Comment (corneadoug) [inline docs/payment_issue.md]: a couple of minutes

**Comment by corneadoug** [comment]
@anthonycorbacho @AhyoungRyu Just proof read

**Comment by AhyoungRyu** [comment]
@corneadoug merci capitaine!

**Comment by anthonycorbacho** [comment]
@AhyoungRyu tu peux merger :)

**Comment by anthonycorbacho** [comment]
@AhyoungRyu sorrrry i totally forgot, can we remove `We are using Stripe as our payments infrastructure. `? this information is not really useful for the user

**Comment by AhyoungRyu** [comment]
@anthonycorbacho Sure. I just redeployed it! Please check it out again.

**Comment by anthonycorbacho** [comment]
@AhyoungRyu look great thank you

**Pull request merged**